client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -447,9 +455,23 @@ class DomainRow extends PureComponent {
 	}
 
 	render() {
-		const { domain, isManagingAllSites, showCheckbox, translate } = this.props;
+		const {
+			domain,
+			contactDetails,
+			isManagingAllSites,
+			site,
+			showCheckbox,
+			purchase,
+			translate,
+		} = this.props;
+		const { showNotice } = this.state;
 		const domainTypeText = getDomainTypeText( domain, translate, domainInfoContext.DOMAIN_ROW );
 		const expiryDate = domain?.expiry ? moment.utc( domain?.expiry ) : null;
+		const { noticeText, statusClass } = resolveDomainStatus( domain, purchase, {
+			siteSlug: site?.slug,
+			getMappingErrors: true,
+			email: contactDetails?.email,
```

Comment:
Sorry I missed one thing, It seems that `contactDetails` info here is always undefined.

Reply:
Nice catch. Yes, the whois request isn't performed 😐 

What we have in this component is the cached value of the contact details (`needsContactDetails` prop retrieves that for us), but I'm not sure if that's useful. 

I have removed it for now, but if that's really needed, we can work on a better solution for it in a follow-up PR. 😬